### ISSUE TITLE
Cinnamon depends on old clutter

### ISSUE THREAD

**Issue by aeonios** [linuxmint/Cinnamon#2951]
I recently tracked down the problem with menus, expo and scale causing freezes and identified clutter as the culprit. I was able to fix it by downgrading to clutter-1.12.2 and clutter-gtk-1.4.2, which cinnamon/muffin are stable with, but those versions are at the bottom of the source tree (ie the oldest versions that are still currently kept around). Also, it seems to me after swapping versions that the desktop was faster and had less tearing with the newer clutter, even though several features were broken by it.

In short, upgrading clutter support would be very nice and probably help with some other goals as well.
